app/javascript/controllers/nested_form_controller.js

```diff
@@ -11,7 +11,7 @@ export default class extends Controller {
 
   removeAssociation(event) {
     event.preventDefault()
-    const item = event.target.closest(".inner-container")
+    const item = event.target.closest(".inner-container") || event.target.closest(".plain-container")
```

Comment:
Can this class to use be passed in as a data- attribute?

Reply:
I did some experiments with using a data-attribute but part of the problem is we still need to select the containing div from the found item using the CSS.

Reply:
@jcoyne howdy. @jermnelson and I looked at this and the Capybara selector comment you raised, when we paired earlier. We didn't find a lightweight way to arrive at deterministic selectors (e.g., email fields with predictable aria labels based on a simple incremented counter) for individual email fields. 

When discussing both of these comments, we realized that they both tie back to the nested-form stimulus controller, and it might be worth copying/pasting that controller to a new contact-email controller that has more appropriate behavior for `addAssociation()` and `removeAssociation()`. That said, if you have something else in mind, I think Jeremy is all ears---but he's also already invested a bunch of time in this PR, so we may be reaching the point when we ought to consider merging this PR and writing issues for the lingering bits we're not happy with. Thoughts?

Reply:
I'm saying pass in the selector as a data- attribute.

Reply:
Updated to now use Stimulus data attribute instead of css class.